src/components/Offerings/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FunctionComponent } from 'react'
+import * as R from 'ramda'
+import styles from './Offering.module.scss'
+import helperStyles from '../helper.module.css'
+import cn from 'classnames'
+import { Col, Row } from 'reactstrap'
+
+export const Offerings: FunctionComponent<{ offerings: Offering[] }> = ({
+  offerings,
+}) => (
+  <div>
+    {R.addIndex<Offering>(R.map)((item, index) => (
+      <OfferingComponent
+        index={index}
+        image={item.image}
+        headline={item.headline}
+        text={item.text}
+      />
```

Comment:
Move this to helper function "mapToComponent"

Reply:
Done